ansible/roles/setup-kafka/defaults/main.yml

```diff
@@ -107,6 +107,19 @@ processing_kafka_overriden_topics:
   - name: ml.observation.druid
     retention_time: 172800000
     replication_factor: 1
+  - name: telemetry.cb.audit
```

Comment:
@shobhit-vashistha These topics are related to Igot. Should this be merged into the Sunbird Core?